lib/Backend/Inline.cpp

```diff
@@ -1928,7 +1950,7 @@ Inline::InlineBuiltInFunction(IR::Instr *callInstr, const FunctionJITTimeInfo *
     IR::Instr *inlineBuiltInEndInstr = nullptr;
     if (inlineCallOpCode ==  Js::OpCode::InlineFunctionApply)
     {
-       inlineBuiltInEndInstr = InlineApply(callInstr, inlineeData, inlinerData, symCallerThis, pIsInlined, profileId, recursiveInlineDepth);
+       inlineBuiltInEndInstr = InlineApply(callInstr, inlineeData, inlinerData, symCallerThis, pIsInlined, profileId, recursiveInlineDepth, inlineCallArgCount - 1);
```

Comment:
Not really. If you look at the calculation of inlineCallArgCount, it is obtained by subtracting 1 from actualCount for built-ins that dont use the this pointer. So, it can be zero. For apply I know that it uses this pointer, so I know that inlineCallArgCount takes into account the 'this' argument. Since, I want to pass the actual arguments passed to .apply, I discount the implicit 'this' argument.

---
In reply to: [85857022](https://github.com/Microsoft/ChakraCore/pull/1838#discussion_r85857022) [](ancestors = 85857022,85823517,85817969)